test/csuite/random_abort/main.c

```diff
@@ -104,25 +141,50 @@ thread_run(void *arg)
     /*
      * Keep a separate file with the records we wrote for checking.
      */
-    (void)unlink(buf);
-    if ((fp = fopen(buf, "w")) == NULL)
-        testutil_die(errno, "fopen");
+    for (i = 0; i < MAX_RECORD_FILES; i++) {
+        (void)unlink(fname[i]);
+        if ((fp[i] = fopen(fname[i], "w")) == NULL)
+            testutil_die(errno, "fopen");
+        /*
+         * Set to line buffering. But that is advisory only. We've seen cases where the result files
+         * end up with partial lines.
+         */
+        __wt_stream_set_line_buffer(fp[i]);
+    }
+
+    testutil_check(td->conn->open_session(td->conn, NULL, NULL, &session));
+
+    /* Make sure that alternative threads operate on column-store table */
+    if (td->id % 2 != 0)
+        columnar_table = true;
+
+    if (columnar_table)
+        testutil_check(session->open_cursor(session, col_uri, NULL, NULL, &cursor));
+    else
+        testutil_check(session->open_cursor(session, uri, NULL, NULL, &cursor));
+
     /*
-     * Set to line buffering. But that is advisory only. We've seen cases where the result files end
-     * up with partial lines.
+     * Initialize the data with predefined data, this buffer modified randomly for the required
+     * buffer to insert.
      */
-    __wt_stream_set_line_buffer(fp);
-    testutil_check(td->conn->open_session(td->conn, NULL, NULL, &session));
-    testutil_check(session->open_cursor(session, uri, NULL, NULL, &cursor));
+    memset(buf, 'a', MAX_VAL - 1);
```

Comment:
Yes, I got your point. I modified the data because of cursor modify operation, I also agree with your point that it should contain the information of key in the data for easier identification. I will add this suggestion to the follow on ticket.

Reply:
Instead of postponing this change, I modified the data as insert-%key as the data whenever this gets modified it is changed to modify-%key, so it will be easy to read the data when it is compared to the first version of aaa...